packages/wallets/src/components/WalletsDisabledAccountsBanner/WalletsDisabledAccountsBanner.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { FC } from 'react';
+import { useWalletAccountsList } from '@deriv/api-v2';
+import { Localize, useTranslations } from '@deriv-com/translations';
+import { SectionMessage, Text, useDevice } from '@deriv-com/ui';
+import './WalletsDisabledAccountsBanner.scss';
+
+type TProps = {
+    disabledAccounts: NonNullable<ReturnType<typeof useWalletAccountsList>['data']>[number][];
+};
+
+const WalletsDisabledAccountsBanner: FC<TProps> = ({ disabledAccounts }) => {
+    const { isDesktop } = useDevice();
+    const { localize } = useTranslations();
+    let disabledAccountsCurrencies = '';
+    const disabledAccountsLength = disabledAccounts.length;
+
+    disabledAccounts.forEach((disabledAccount, index) => {
+        if (disabledAccountsLength > 1) {
+            if (index === disabledAccountsLength - 1) {
+                disabledAccountsCurrencies += ` and ${disabledAccount.currency}`;
+            } else {
+                disabledAccountsCurrencies += `${disabledAccount.currency}, `;
+            }
+        } else {
+            disabledAccountsCurrencies += disabledAccount.currency;
+        }
+    });
+
+    return (
+        <div className='wallets-disabled-account-banner__container'>
+            <SectionMessage variant='warning'>
+                <Text lineHeight='lg' size={isDesktop ? 'sm' : 'md'}>
```

Comment:
For new components can we add alignment to the `Text` components used to cater for RTL, as they have left alignment by default
```suggestion
                <Text align='start' lineHeight='lg' size={isDesktop ? 'sm' : 'md'}>
```